models/entities/person.go

```diff
@@ -80,7 +84,45 @@ func NewPerson(store *Store, id string) (*Person, error) {
 		return nil, err
 	}
 
-	// Revisit: Create Inbox/Outbox/etc
+	// TOOD: These hould be Inbox/Outbox kinds but this will do For Now (TM)
+	inbox, err := NewStream(store, id+"/inbox")
+	if err != nil {
+		return nil, errors.Wrap(err, "Creating inbox")
+	}
+
+	outbox, err := NewStream(store, id+"/outbox")
+	if err != nil {
+		return nil, errors.Wrap(err, "Creating outbox")
+	}
+
+	inbox.AttributedTo = jsonld.ToRef(id)
+	outbox.AttributedTo = jsonld.ToRef(id)
+	obj.Inbox = jsonld.ToRef(inbox.GetID())
+	obj.Outbox = jsonld.ToRef(outbox.GetID())
```

Comment:
Could we maybe make a `SetInbox()` and `SetOutbox` method pair for this? It feels like somewhere down the line, we will forget to set either `obj.{In,Out}box` or `{in,out}box.AttributedTo`.

Reply:
This sounds like a good idea

Actually I'm tempted to make accessors (GetInbox, GetOutbox?) which lazily create inboxes/outboxes?

Reply:
I like this idea, especially if you do both of these things